src/guides/v2.4/config-guide/prod/config-reference-systemxml.md

```diff
@@ -359,28 +359,37 @@ The following validation rules are available:
-| `validate-zip-us`               | Enter a valid ZIP code. For example, 90602 or 90602-1234.                                                               |
-| `validate-date-au`              | Use this date format: dd/mm/yyyy. For example, 17/03/2006 for the 17th of March, 2006.                                  |
-| `validate-currency-dollar`      | Enter a valid $ amount. For example, $100.00.                                                                           |
+| `time`                          | Allows a valid time in 24-hour format, between 00:00 and 23:59. For example `15`, `15:05` or `15:05:48`.                |
+| `time12h`                       | Allows a valid time in 12-hour format, between 12:00 am and 11:59:59 pm. For example `3 am`, `11:30 pm`, `02:15:00 pm`. |
+| `mobileUK`                      | Allows a (UK) mobile phone number.                                                                                      |
+| `phoneUK`                       | Allows a (UK) phone number.                                                                                             |
+| `phoneUS`                       | Allows a (US) phone number.                                                                                             |
+| `required-entry`                | Disallows an empty value (equivalent validation as `validate-no-empty`).<br>Validation failure message: "This is a required field." |
+| `validate-no-html-tags`         | Disallows usage of HTML tags.                                                                                           |
+| `validate-select`               | Enforces a that the select option chosen not have a `null` value, string value of `none`, or string length of 0.        |
+| `validate-no-empty`             | Disallows an empty value (equivalent validation as `requried-entry`).<br>Validation failure message: "Empty value."     |
+| `validate-alphanum-with-spaces` | Allows usage of letters (a-z or A-Z), numbers (0-9), or spaces only.                                                    |
+| `validate-data`                 | Allows usage of letters (a-z or A-Z), numbers (0-9), or underscores (\_) only.<br>The first character must be a letter.<br>(Must match expression: `/^[A-Za-z]+[A-Za-z0-9_]+$/`)<br>Validation failure message: "Please use only letters (a-z or A-Z), numbers (0-9) or underscore (\_) in this field, and the first character should be a letter."  |
+| `validate-street`               | Allows usage of letters (a-z or A-Z), numbers (0-9), spaces, and “#” only.                                              |
+| `validate-phoneStrict`          | Allows a valid phone number. For example, (123) 456-7890 or 123-456-7890.                                               |
+| `validate-phoneLax`             | Allows a valid phone number. For example, (123) 456-7890 or 123-456-7890.                                               |
```

Comment:
What is the difference between Strict & Lax?

Reply:
Good question. I remember these existing in M1 as well. off the top of my head I don't know but I can get back to you on this.